packages/server/database/migrations/20200505120000-timelineEventIsActive.ts

```diff
@@ -0,0 +1,35 @@
+import {R} from 'rethinkdb-ts'
+
+export const up = async function(r: R) {
+  try {
+    await r
+      .table('TimelineEvent')
+      .update({
+        isActive: true
+      })
+      .run()
+
+    await r
+      .table('TimelineEvent')
+      .indexCreate('meetingId')
+      .run()
+  } catch (e) {
+    console.log(e)
+  }
+}
+
+export const down = async function(r: R) {
+  try {
+    await r
+      .table('TimelineEvent')
+      .replace((row) => row.without('isActive'))
+      .run()
+
+    await r
+      .table('TimelineEvent')
+      .indexCreate('meetingId')
```

Comment:
I may be misunderstanding something, but wouldn’t we want to remove the index here?

Reply:
Would we use `indexDrop('meetingId')`?

Reply:
great catch!